src/environmentd/src/lib.rs

```diff
@@ -387,11 +388,11 @@ pub async fn serve(config: Config) -> Result<Server, anyhow::Error> {
     });
 
     // Start telemetry reporting loop.
-    if let (Some(segment_client), Some(frontegg)) = (segment_client, config.frontegg) {
+    if let Some(segment_client) = segment_client {
```

Comment:
Why don't we care about frontegg anymore?

Reply:
Ah, because previously the frontegg config was the easiest place to get the org ID from. Now we are smart enough to pull it out of the env ID.